website/docs/docs/build/sl-getting-started.md

```diff
@@ -4,23 +4,33 @@ title: Get started with MetricFlow
 description: "Learn how to create your first semantic model and metric."
 sidebar_label: Get started with MetricFlow
 tags: [Metrics, Semantic Layer]
+meta:
+  api_name: dbt Semantic Layer API
 ---
 
-This getting started page recommends a workflow to help you get started creating your first metrics. Here are the following steps you'll take:
+This getting started page recommends a workflow to help you create your first metrics. This guide uses the [Jaffle shop example project](https://github.com/dbt-labs/jaffle-sl-template). If you prefer using a sandbox project to follow the guide, you can clone the repo from GitHub. Otherwise, you can create semantic models and metrics for your own dbt project. 
```

Comment:
Maybe with the ongoing discussions around the semantics of `guide` we should avoid the term here. I rewrote a portion to accommodate its removal. Let me know what you think. 
```suggestion
This getting started page presents a sample workflow to help you create your first metrics. It uses the [Jaffle shop example](https://github.com/dbt-labs/jaffle-sl-template) as the project data source and is available for you to use. If you prefer, you can create semantic models and metrics for your own dbt Cloud project. 
```